src/hotspot/share/runtime/javaThread.hpp

```diff
@@ -166,14 +166,14 @@ class JavaThread: public Thread {
   // ID used as owner for inflated monitors. Same as the j.l.Thread.tid of the
   // current _vthread object, except during creation of the primordial and JNI
   // attached thread cases where this field can have a temporary value.
-  int64_t _lock_id;
+  int64_t _monitor_owner_id;
 
  public:
-  void set_lock_id(int64_t tid) {
-    assert(tid >= ThreadIdentifier::initial() && tid < ThreadIdentifier::current(), "invalid tid");
-    _lock_id = tid;
+  void set_monitor_owner_id(int64_t val) {
+    assert(val >= ThreadIdentifier::initial() && val < ThreadIdentifier::current(), "");
+    _monitor_owner_id = val;
```

Comment:
Nit: Using `id`  rather than `val` would be more consistent with other changes (`ObjectMonitor::owner_id_from`)

Reply:
Fixed.